dnngior/build_model.py

```diff
@@ -46,74 +46,77 @@ def refine_model(gpfilledModel, draftModel=None, scale = 1000, unscalled = None)
         a model that is ready to use
 
     '''
-    
+
     mc = gpfilledModel.copy()
     if unscalled is None:
         unscalled=[]
-    
-    for metab in mc.metabolites:
-
-        metabName = metab.id.split('_')[0]
-        metabSufix = metab.id.split('_')[1]
-        
-        if metabName in compounds_dict:
-        
-            # 1) Add metabolite info
-            
-            if metabName in compounds_aliases_dict:
-                metab.annotation = compounds_aliases_dict[metabName].copy()
+    if dbType=='ModelSEED':
+        for metab in mc.metabolites:
+
+            metabName = metab.id.split('_')[0]
+            metabSufix = metab.id.split('_')[1]
+
+            if metabName in compounds_dict:
+
+                # 1) Add metabolite info
+
+                if metabName in compounds_aliases_dict:
+                    metab.annotation = compounds_aliases_dict[metabName].copy()
```

Comment:
I guess this is related to #24 right? 
If yes, then it would be a good thing to edit your PR message and mention that the issue is addressed in this PR, to be able to track it!  :wink: